executor/runtime/docker/docker_linux.go

```diff
@@ -67,9 +67,16 @@ var systemServices = []serviceOpts{
 		required:     false,
 	},
 	{
-		humanName: "atlas",
-		unitName:  "atlas-titus-agent",
-		required:  false,
+		humanName:    "atlas",
+		unitName:     "titus-atlasd",
+		enabledCheck: shouldStartAtlasd,
+		required:     false,
+	},
+	{
+		humanName:    "atlas",
```

Comment:
I've thought about removing the humanName, I don't think we need it and can just use the unitname. Would you like it to be distinct from the other atlasd so you can tell easily from the logs which one it is talking about?

Reply:
I intend to wrap this up, then clean up all the sidecars as per the RFC. Can roll removing humanName into that pull request.